src/main/java/HW_7/Square.java

```diff
@@ -0,0 +1,20 @@
+package HW_7;
+
+public class Square extends Figure{
+    private float side;
+    Square(float side) {
+        this.side = side;
+    }
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+    @Override
+    public double getPerimeter() {
+        return side * 4;
+    }
+    @Override
+    public String toString() {
+        return "Square " + super.toString();
+    }
```

Comment:
please add extra lines between methods.

use Ctrl+ alt + l

Reply:
fixed

Reply:
not fixed

Reply:
fixed